### ISSUE TITLE
Failing to received Bungee channel messages

### ISSUE THREAD

**Issue by idkw** [Puharesource/TitleManager#62]
I have several plugins that are using the Bungee channel and one of them (HolographicDisplay) makes  TitleManager fails to handle these messages.

I post the issue here because whereas it's HolographicDisplay fault from sending corrupted messages, it's TitleManager's responsibility to ignore these messages and not to crash because of them.

Here is the error log :  

```

[16:23:49 WARN]: Could not pass incoming plugin message to TitleManager v1.5.11
java.lang.IllegalStateException: java.io.EOFException
    at com.google.common.io.ByteStreams$ByteArrayDataInputStream.readUTF(ByteStreams.java:449) ~[spigot-1.8.8.jar:git-Spigot-d97e08b-5e85956]
    at io.puharesource.mc.titlemanager.backend.bungee.BungeeManager.onPluginMessageReceived(BungeeManager.java:87) ~[?:?]
    at org.bukkit.plugin.messaging.StandardMessenger.dispatchIncomingMessage(StandardMessenger.java:427) [spigot-1.8.8.jar:git-Spigot-d97e08b-5e85956]
    at net.minecraft.server.v1_8_R3.PlayerConnection.a(PlayerConnection.java:2113) [spigot-1.8.8.jar:git-Spigot-d97e08b-5e85956]
    at net.minecraft.server.v1_8_R3.PacketPlayInCustomPayload.a(SourceFile:55) [spigot-1.8.8.jar:git-Spigot-d97e08b-5e85956]
    at net.minecraft.server.v1_8_R3.PacketPlayInCustomPayload.a(SourceFile:8) [spigot-1.8.8.jar:git-Spigot-d97e08b-5e85956]
    at net.minecraft.server.v1_8_R3.PlayerConnectionUtils$1.run(SourceFile:13) [spigot-1.8.8.jar:git-Spigot-d97e08b-5e85956]
    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_05]
    at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_05]
    at net.minecraft.server.v1_8_R3.SystemUtils.a(SourceFile:44) [spigot-1.8.8.jar:git-Spigot-d97e08b-5e85956]
    at net.minecraft.server.v1_8_R3.MinecraftServer.B(MinecraftServer.java:714) [spigot-1.8.8.jar:git-Spigot-d97e08b-5e85956]
    at net.minecraft.server.v1_8_R3.DedicatedServer.B(DedicatedServer.java:374) [spigot-1.8.8.jar:git-Spigot-d97e08b-5e85956]
    at net.minecraft.server.v1_8_R3.MinecraftServer.A(MinecraftServer.java:653) [spigot-1.8.8.jar:git-Spigot-d97e08b-5e85956]
    at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:556) [spigot-1.8.8.jar:git-Spigot-d97e08b-5e85956]
    at java.lang.Thread.run(Thread.java:745) [?:1.8.0_05]
Caused by: java.io.EOFException
    at java.io.DataInputStream.readUnsignedShort(DataInputStream.java:340) ~[?:1.8.0_05]
    at java.io.DataInputStream.readUTF(DataInputStream.java:589) ~[?:1.8.0_05]
    at java.io.DataInputStream.readUTF(DataInputStream.java:564) ~[?:1.8.0_05]
    at com.google.common.io.ByteStreams$ByteArrayDataInputStream.readUTF(ByteStreams.java:447) ~[spigot-1.8.8.jar:git-Spigot-d97e08b-5e85956]
    ... 14 more

```

Thank you.

**Comment by Puharesource** [comment]
Just to clarify, this doesn't "crash" the plugin, it simply spams your console (Which of course isn't acceptable). I'll fix this in version 1.6.0, though that might take a while, due to the fact that I've had a ton of work to do for the past month or two.

**Comment by idkw** [comment]
Yes I meant it crashes the thread handling the bungee messages.

Anyway thank you for your work on this plugin.

**Comment by Puharesource** [comment]
I'm glad you like it :)

**Comment by xion87** [comment]
same issue here with bungeeplayer list 
http://hastebin.com/ugiganivaq.avrasm

**Comment by 7h3ev1l** [comment]
I hope this will be fixed soon... i use holo display too!!!

**Comment by Puharesource** [comment]
Is this still a problem with version 2.0.0? The pre-release for 2.0.0 can be found here: https://github.com/Puharesource/TitleManager/releases/tag/2.0.0-12

**Comment by Puharesource** [comment]
Seems to be fixed with 2.0
